### PR TITLE
Email update

### PR DESCRIPTION
**Pull request by simonnebMOJ**
Updates the the one-time code email template - changes include:
* new crown logo
* new GOV.UK banner name to replace HMPPS
* removal of superfluous text

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/sendlegalmailtoprisonsapi/onetimecode/OneTimeCodeEmailSender.kt [hunk 1]
```diff
@@ -54,13 +52,13 @@ class OneTimeCodeEmailSender(
                             <tr>
                                 <td style="font-family:Helvetica,Arial,sans-serif;padding:4px 0;border-bottom:solid 10px #005ea5" width="30">
 
-                                    <a href="https://www.gov.uk/" title="Go to the GOV.UK homepage" style="font-size:24px;font-weight:bold;text-decoration:none;color:#ffffff" target="_blank" data-saferedirecturl="https://www.google.com/url?q=https://www.gov.uk/&amp;source=gmail&amp;ust=1648555909470000&amp;usg=AOvVaw2uN4hTGDbOHXGNCBJLkYyo">
-                                        <img src="cid:crest.png" alt="" style="margin:6px 6px 0 20px" class="CToWUd" width="30">
+                                    <a href="https://www.gov.uk/" title="Go to the GOV.UK" style="font-size:24px;font-weight:bold;text-decoration:none;color:#ffffff" target="_blank" data-saferedirecturl="https://www.google.com/url?q=https://www.gov.uk/&amp;source=gmail&amp;ust=1648555909470000&amp;usg=AOvVaw2uN4hTGDbOHXGNCBJLkYyo">
```
Comment (dhirajshettymoj) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/sendlegalmailtoprisonsapi/onetimecode/OneTimeCodeEmailSender.kt:55]: Go to GOV.UK?

File: src/main/kotlin/uk/gov/justice/digital/hmpps/sendlegalmailtoprisonsapi/onetimecode/OneTimeCodeEmailSender.kt [hunk 2]
```diff
@@ -54,13 +52,13 @@ class OneTimeCodeEmailSender(
...
+                                        <img src="cid:crest.png" alt="" style="margin:10px 30px 10px 20px" class="CToWUd" width="30">
                                     </a>
 
                                 </td>
                                 <td style="font-family:Helvetica,Arial,sans-serif;padding:4px 0;border-bottom:solid 10px #005ea5">
-                                    <a href="https://www.gov.uk/" title="Go to the GOV.UK homepage" style="font-size:24px;font-weight:bold;text-decoration:none;color:#ffffff" target="_blank" data-saferedirecturl="https://www.google.com/url?q=https://www.gov.uk/&amp;source=gmail&amp;ust=1648555909470000&amp;usg=AOvVaw2uN4hTGDbOHXGNCBJLkYyo">HMPPS</a>
+                                    <a href="https://www.gov.uk/" title="Go to the GOV.UK" style="font-size:24px;font-weight:bold;text-decoration:none;color:#ffffff" target="_blank" data-saferedirecturl="https://www.google.com/url?q=https://www.gov.uk/&amp;source=gmail&amp;ust=1648555909470000&amp;usg=AOvVaw2uN4hTGDbOHXGNCBJLkYyo">GOV.UK</a>
```
Comment (dhirajshettymoj) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/sendlegalmailtoprisonsapi/onetimecode/OneTimeCodeEmailSender.kt:61]: same as above

**Review by dhirajshettymoj — COMMENTED** [review]
2 minor review comments around text

File: src/main/kotlin/uk/gov/justice/digital/hmpps/sendlegalmailtoprisonsapi/onetimecode/OneTimeCodeEmailSender.kt (hunk 1 above)
Comment (simonnebMOJ) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/sendlegalmailtoprisonsapi/onetimecode/OneTimeCodeEmailSender.kt]: Updated

File: src/main/kotlin/uk/gov/justice/digital/hmpps/sendlegalmailtoprisonsapi/onetimecode/OneTimeCodeEmailSender.kt (hunk 2 above)
Comment (simonnebMOJ) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/sendlegalmailtoprisonsapi/onetimecode/OneTimeCodeEmailSender.kt]: Updated

**Review by dhirajshettymoj — APPROVED** [review]
LGTM

**Pull request merged**